res/mkn/clang.yaml

```
@@ -0,0 +1,21 @@
+env: |
+  OMPI_CXX=clang++-17
+  PATH=/opt/mpi/cuda/bin:${PATH}
+compiler:
+  mask:
+    g++:
+      mpicxx
+file:
+  - type: cpp:cxx:cc
+    archiver: ar -cr
+    compiler:  mpicxx ${cxx.flags} 
+    linker:  mpicxx ${lnk.both}
```

yamllint (1.29.0-1)

[warning] too many spaces after colon

(colons)